src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,102 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.exeption.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final StorageDao storageDao = new StorageDaoImpl();
+    private static final RegistrationService registrationService = new RegistrationServiceImpl();
+    private static final Integer DEFAULT_AGE = 18;
+    private static final String DEFAULT_PASSWORD = "123456";
+    private static final String DEFAULT_LOGIN = "Ukraine";
+    private User defaultUser;
+
+    @BeforeEach
+    void setUp() {
+        defaultUser = new User();
+        defaultUser.setAge(DEFAULT_AGE);
+        defaultUser.setPassword(DEFAULT_PASSWORD);
+        defaultUser.setLogin(DEFAULT_LOGIN);
+    }
+
+    @AfterEach
+    void clearAfterTest() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void ageElementIsNull_NotOk() {
```

Comment:
Rename please test methods according to the task description:

There are a lot of ways to name your test methods. The main point is that they should have informative names and be consistent along with other developers in your team. For this task use such convention: <methodUnderTest>_<state>_<expectedBehavior>; For example, if we are testing the method register with a null
user's age the test method name should be register_nullAge_notOk. notOk is because the test expects the register method to throw an exception.